sys/xtimer/xtimer_core.c

```diff
@@ -178,12 +180,27 @@ int _xtimer_set_absolute(xtimer_t *timer, uint32_t target)
     uint32_t now = _xtimer_now();
```

Comment:
I know it's not you who wrote this, but would you mind documenting what this function is supposed to do?

Without knowing the expected behaviour it is hard to see the fix.

Reply:
If you meant for `_xtimer_set_absolute` done.